target/linux/ath79/dts/qca95xx_dlink_dap-2xxx.dtsi

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&spi {
+	status = "okay";
+
+	num-cs = <1>;
```

Comment:
num-cs can probably be dropped.

Reply:
I'll just leave this here as I have no account on patchwork:
Some ath79 devices also still have `cs-gpios = <0>, <0>;` along with num-cs, which should be removed as well (no idea why someone would declare two chipselects on the same gpio).

Reply:
That's a different subject, but you could patch that out as well in a separate patch if you like.